recipe/run_test.py

```diff
@@ -6,6 +6,9 @@
     from nose import run_exit
     from traits.etsconfig.api import ETSConfig
     import os
+    from subprocess import call
+    call("python setup.py build_ext --inplace",
```

Comment:
I came to this conclusion as I read source of scikit-learn now:
https://github.com/scikit-learn/scikit-learn/blob/master/build_tools/circle/build_doc.sh#125
https://github.com/scikit-learn/scikit-learn/blob/master/build_tools/travis/install.sh#L117

I wish I had learned about this earlier...